### PR TITLE
TYPO3 13 support

### PR DESCRIPTION
**Pull request by ulrichmathes**
I'm not sure about [[v13] translate without alternativeLanguageKeys](https://github.com/sitegeist/fluid-components/commit/6dd1ec617b278b75115bcda938cd453f388e6687) as it seems we could get rid of the `alternativeLanguageKeys` as the core does. But this would have a higher impact and would need preparation beforehand. 

I marked the ViewHelper Arguments as deprecated and in doing so preparing for the next major release? We planned to give the extension a fresh-up within this year. So that could be a better case to get rid of this parameter.

### CODE REVIEW

File: ext_emconf.php [hunk 1]
```diff
@@ -7,13 +7,11 @@
...
     'state' => 'stable',
-    'uploadfolder' => false,
-    'clearCacheOnLoad' => false,
     'version' => '',
     'constraints' => [
         'depends' => [
-            'typo3' => '11.5.0-12.9.99',
-            'php' => '7.4.0-8.9.99'
+            'typo3' => '12.4.0-13.4.99',
+            'php' => '8.2.0-8.2.99'
```
Comment (s2b) [inline ext_emconf.php:14]: probably at least `8.2.0-8.3.99`

**Review by s2b — COMMENTED** [review]
Other than the small comment, this looks fine to me. I haven't tested it though, only did code review.

I would agree with the strategy to make additional language keys work for now, but to deprecate it and to remove it with v4.

File: ext_emconf.php (hunk 1 above)
Comment (ulrichmathes) [reply ext_emconf.php:14]: argh, ext_emconf.php - maybe we should generate it completely from composer.json instead of setting only version in CI.

**Review by rafaelheid-it — APPROVED** [review]
Code reviewed this and it looks fine to me. Also seems functional - components render as they should.

**Comment by ulrichmathes** [comment]
These changes are only compatible with fluid 2.14.1. Along with TYPO3 13.3, there will be a major update of fluid. So these changes will be released as a pre-release with hard version coupling to TYPO3 13.2 and fluid 2.14.1.

Steps forward to TYPO3 13.4 and fluid 4 will follow after Feature Freeze of TYPO3.

**Pull request merged**